packages/widgetbook_generator/README.md

```diff
@@ -0,0 +1,62 @@
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/widgetbook/widgetbook/ci?style=flat-square)
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/widgetbook/widgetbook/ci?label=test&style=flat-square)
+
+# Introduction 
+
+[package:widgetbook_generator](https://pub.dev/packages/widgetbook_generator) allows to generate and setup code for [package:widgetbook](https://pub.dev/packages/widgetbook) which makes using and maintaining Widgetbook easier.
+
+## Before you start
+
+We recommend reading the documentation of [package:widgetbook](https://pub.dev/packages/widgetbook) before you start with this package. 
+
+# Packages 
+
+This package requires the following dependencies: 
+
+| Package           | Pub |
+| ----------------- | --------------------------------- |
+| [package:widgetbook](https://pub.dev/packages/widgetbook) | [![Pub Version](https://img.shields.io/pub/v/widgetbook?style=flat-square)](https://pub.dev/packages/widgetbook) |
+| [package:widgetbook_annotation](https://pub.dev/packages/widgetbook_annotation) | [![Pub Version](https://img.shields.io/pub/v/widgetbook_annotation?style=flat-square)](https://pub.dev/packages/widgetbook_annotation) |
+
+and the following dev dependencies:
+
+| Package           | Pub |
+| ----------------- | --------------------------------- |
+| [package:widgetbook_generator](https://pub.dev/packages/widgetbook_generator) | [![Pub Version](https://img.shields.io/pub/v/widgetbook_generator?style=flat-square)](https://pub.dev/packages/widgetbook_generator) |
+| [package:build_runner](https://pub.dev/packages/build_runner) | [![Pub Version](https://img.shields.io/pub/v/build_runner?style=flat-square)](https://pub.dev/packages/build_runner) |
+
+The `pubspec.yaml` file could look like this:
+
+```
+dependencies:
+  widgetbook:
+  widgetbook_annotation:
+
+dev_dependencies:
+  build_runner:
+  widgetbook_generator:
+```
+
+# Annotations
+
+The generator works with the annotations documented at [package:widgetbook_annotation](https://pub.dev/packages/widgetbook_annotation). 
+
+# Run the generator
+
+The generator defined in this package uses [package:build_runner](https://pub.dev/packages/build_runner) to generate the desired code.
+
+You can execute the generator by running `flutter pub run build_runner build`. 
+
+If you are making a lot of changes while developing you can also run `flutter pub run build_runner watch` so [package:build_runner](https://pub.dev/packages/build_runner) will listen for changes in the file system and update Widgetbook accordingly.
+
+# Start the generated app
+
+The generator will create the `app.widgetbook.dart` file in the same directory as the file in which `@WidgetbookApp` is used. For details about this see the documentation of [package:widgetbook_annotation](https://pub.dev/packages/widgetbook_annotation).
```

Comment:
here you are using @ in front of the annotation. I like it better that way. We should do that consistently

Reply:
For details about this, see